cdm/core/src/test/java/ucar/nc2/ncml/TestEnhanceClassifier.java

```diff
@@ -112,4 +118,27 @@ public void testEnhanceClassifier_integers() throws IOException {
     }
 
   }
+
+  @Test
+  public void testEnhanceClassifier_classification() throws IOException {
+
+    try (NetcdfFile ncfile = NetcdfDatasets.openDataset(dataDir + "testAddToClassifier.ncml", true, null)) {
+
+      Variable Classify_Specsx = ncfile.findVariable("class_specs");
```

Comment:
Minor style issue: would use lower camel case for the variable name, something like:
```suggestion
      Variable classifySpecs = ncfile.findVariable("class_specs");
```